src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task{
+    protected String at;
+
+
```

Comment:
Should this extra line break be here?

Reply:
Oh yeah, I seem to have forgotten a line break!